Description: Mark tests that may fail with pyproj built with PROJ < 8.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/lib/cartopy/tests/test_crs.py
+++ b/lib/cartopy/tests/test_crs.py
@@ -82,10 +82,12 @@ class TestCRS:
         r_inverted = np.array(ll.transform_point(r_east, r_north, osgb))
         assert_arr_almost_eq(r_inverted, [lon, lat])
 
+    @pytest.mark.xfail(strict=False)
     @pytest.mark.parametrize('approx', [True, False])
     def test_osgb(self, approx):
         self._check_osgb(ccrs.OSGB(approx=approx))
 
+    @pytest.mark.xfail(strict=False)
     def test_epsg(self):
         uk = ccrs.epsg(27700)
         assert uk.epsg_code == 27700
